src/components/home/WhoAreWe.tsx

```diff
@@ -1,5 +1,43 @@
+import Image from "next/image";
+import groupHome from "@/public/home/groupHome.webp";
+import Button from "@/components/Button";
+
 const WhoAreWe = () => {
-  return <div>Who Are We?</div>;
+  return (
+    <div className="relative h-[800px] w-full">
+      <Image
+        src={groupHome}
+        alt="groupHome"
+        className="absolute h-full w-full object-cover opacity-80"
+      />
+      <div className="bg-naama-ivory-100 absolute inset-0 opacity-80" />
+
+      <div className="relative z-10 flex h-full w-full items-center justify-center gap-20">
+        <h1 className="text-naama-blue-200 font-playfair text-8xl">
+          <span className="underline">Who</span> are
+          <span className="italic"> We</span>?
+        </h1>
+        <div className="flex-col px-60">
+          <p className="font-nunito text-naama-blue-200 max-w-5xl pb-10 text-4xl">
```

Comment:
```suggestion
        <div className="flex flex-col gap-4">
          <p className="font-nunito text-naama-blue-200 p-10 text-xl">
```

flex-col requires flex first